### PR TITLE
Add Contributor Covenant

### PR DESCRIPTION
**Pull request by garyrussell**
Also convert CONTRIBUTING from markdown to asciidoc.

### CODE REVIEW

**Comment by garyrussell** [comment]
@artembilan Please give this some priority when you return, particularly the `CONTRIBUTING.adoc` conversion.

We don't currently have that in spring-AMQP so I plan to port both files there after this is merged.

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -147,27 +177,31 @@ Please carefully follow the whitespace and formatting conventions already presen
...
+still reads
 
-```java
+[source java]
+----
  * Copyright 2002-2011 the original author or authors.
-```
+----
 
 then be sure to update it to 2013 appropriately
```
Comment (artembilan) [inline CONTRIBUTING.adoc]: Have missed to fix this as well.
Do that on merge.

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -26,13 +36,13 @@ Once you've completed the web form, simply add the following in a comment on you
     I have signed and agree to the terms of the SpringSource Individual
     Contributor License Agreement.
```
Comment (artembilan) [inline CONTRIBUTING.adoc]: Looks like we can keep all the words on a single line.
Any objections do not do that on merge?

Yes, minor, of course, but at a glance I would use this sentence in my PR as is: with that extra new line.

**Comment by garyrussell** [comment]
OK; thanks.

File: CONTRIBUTING.adoc [hunk 3]
```diff
@@ -129,7 +158,8 @@ Please carefully follow the whitespace and formatting conventions already presen
 
 ## Add Apache license header to all new classes
 
-```java
+[source, java]
+----
 /*
  * Copyright 2013 the original author or authors.
```
Comment (artembilan) [inline CONTRIBUTING.adoc]: I think we should fix this as well.

**Comment by artembilan** [comment]
That's all.
Merging with mentioned and agreed fixes.

**Comment by garyrussell** [comment]
Thanks

**Comment by artembilan** [comment]
Merged as https://github.com/spring-projects/spring-integration/commit/d91a6fd4670892a5b6d5e6ad49d4e33a21409d78.

Let me know if we should add `CoC` into all our projects. Not only Spring AMQP yet.

**Pull request closed without merging**